egs2/TEMPLATE/vc1/sid/README.md

```diff
@@ -0,0 +1 @@
+The scripts in this directory were copied from Kaldi-ASR, https://github.com/kaldi-asr/kaldi, and their licenses follow the original license, https://github.com/kaldi-asr/kaldi/blob/master/COPYING, of Kaldi.
```

Comment:
you may use symlink for these. I'm also not sure if we would want to keep this as default, as Kaldi's version is quite old (where I think it might be better to keep it as an option instead of the default one). Also due to the reason that our ESPnet version would be on quite soon.